src/core.ts

```diff
@@ -116,8 +116,8 @@ export function instrument(eventState: IInstrumentationEventState) {
 }
 
 function getUnixTimestamp(): number {
-  return (window.performance && typeof performance.now === "function")
-    ? performance.now() + performance.timing.navigationStart
+  return (window.performance && window.performance.now)
```

Comment:
No single strong reason, but:
1. I feel like it's redundant. It can never be anything else but a function (unless manually overriden by someone)
2. I've looked at other projects and how they deal with performance safety checks - vast majority just checks that performance.now exists
3. We've seen errors such as 'object 'performance' has no method 'now'', however every single use of performance.now in our code base was wrapped in this condition. So while i'm not sure how this error is possible in the first place, I decided to tweak a condition and see if it makes a difference.